.github/ISSUE_TEMPLATE/a_issue_report.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Issues
+about: Create an issue if you need any help
+title: '[Issue] '
+labels: 'help wanted, question'
+assignees: ''
+
+---
+
```

Comment:
Add:
**Did you look for other closed issues that have the same problem?**=
So we don't get bonked by duplicate issues

Reply:
https://github.com/stevemao/github-issue-templates/blob/master/no-duplicates/ISSUE_TEMPLATE.md

Something like this on top?

Reply:
sure